sklearn/linear_model/_stochastic_gradient.py

```diff
@@ -1789,14 +1789,15 @@ class SGDRegressor(BaseSGDRegressor):
 
     alpha : float, default=0.0001
         Constant that multiplies the regularization term. The higher the
-        value, the stronger the regularization.
-        Also used to compute the learning rate when set to `learning_rate` is
-        set to 'optimal'.
+        value, the stronger the regularization. Also used to compute the
+        learning rate when `learning_rate` is set to 'optimal'.
+        Values must be in the range `[0.0, inf)`.
 
     l1_ratio : float, default=0.15
         The Elastic Net mixing parameter, with 0 <= l1_ratio <= 1.
         l1_ratio=0 corresponds to L2 penalty, l1_ratio=1 to L1.
         Only used if `penalty` is 'elasticnet'.
+        Values must be in the range `[0.0, 1.0]`.
```

Comment:
Unsure about adding this to all parameters. It seems useful in some cases but in others it is less clear to me if it adds information or just states "the obvious" (but then what is obvious depends on who you are..)

Reply:
> Unsure about adding this to all parameters. It seems useful in some cases but in others it is less clear to me if it adds information or just states "the obvious" (but then what is obvious depends on who you are..)

@betatim Thanks! I don't feel strongly about it other than making it consistent with what we see in [`SGDClassifier`](https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.SGDClassifier.html).

I have an additional question. This is my first time creating a pull request in an open-source project. You just approved this pull request. Is there any additional action required on my end or will someone merge this eventually?

Reply:
Let's keep the ranges in the doc strings then!